.github/workflows/haskell-ci.yml

```diff
@@ -0,0 +1,228 @@
+# This GitHub workflow config has been generated by a script via
+#
+#   haskell-ci 'github' 'cabal.project'
```

Comment:
haskell-ci --config=cabal.haskell-ci cabal.project

Reply:
Oh. I see that also fixes the duplicate work problem :+1: